packages/manager/apps/dedicated/client/app/ip/components/list/firewall/ip-ip-firewall-rule.service.js

```diff
@@ -124,6 +124,18 @@ export default /* @ngInject */ function IpFirewallService(
       );
   };
 
+  this.sortList = function sortList(listToSort) {
+    return listToSort.sort((a, b) => {
+      if (a.name < b.name) {
+        return -1;
+      }
+      if (a.name > b.name) {
+        return 1;
+      }
+      return 0;
+    });
```

Comment:
```suggestion
return listToSort.sort((a, b) => a.name.localeCompare(b.name));
```

Reply:
Sorry but your suggestion doesn't work.

Reply:
I checked the code and it's because name is an integer, not a string.
In this case this should work :
```suggestion
    return listToSort.sort((a, b) => (a.name > b.name));
```